core/src/main/java/io/questdb/std/datetime/microtime/TimestampFormatUtils.java

```diff
@@ -452,4 +452,12 @@ private static long parseTimestamp(CharSequence value, int lo, int hi) throws Nu
 
     public static void init() {
     }
+
+    public static void main(String[] args) throws NumericException {
```

Comment:
Looks like a debug leftover, isn't it?

Reply:
fixed in https://github.com/questdb/questdb/pull/2709/commits/cd6f969c9403b9497dcdb0f36d6db6fd9f032bb2